pkgs/by-name/up/uppaal/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  makeWrapper,
+  jdk17,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook,
```

Comment:
Seems like ofborg is complaining about this, recommending to use either `wrapGAppsHook3` or `wrapGAppsHook4`. Not sure which to choose, but they are used for GTK 3 and GTK 4 respectively. Maybe you know?

Reply:
> Seems like ofborg is complaining about this, recommending to use either `wrapGAppsHook3` or `wrapGAppsHook4`. Not sure which to choose, but they are used for GTK 3 and GTK 4 respectively. Maybe you know?

I am not sure, but since Uppaal is very mature software, I find it hard to believe it supports GTK4... It is only relevant when selecting the GTK+ theme in Uppaal, so it is not crucial to the core functionality anyway.

I tested with wrapGAppsHook, and I did not get any warnings, so I believe it is fine.

Thanks for your help!